examples/11_advanced_parametric_study.py

```diff
@@ -27,7 +27,12 @@
 You perform a parametric study if you want to optimize additive machine parameters
 to achieve a specific result. Here, the :class:`ParametricStudy` class is used to
 conduct a parametric study. While not essential, the :class:`ParametricStudy`
-class provides data management and visualization features that make the work easier.
+class provides data management features that make the work easier. Also, the
+``ansys.additive.widgets`` package can be used to create interactive user
```

Comment:
The term "interactive user interfaces" seems redundant. Do you want to say "interactive visualizations"? But maybe the package allows you to do more than just visualization. 
I don't feel strongly about this.